agent/lib/kontena/workers/container_health_check_worker.rb

```diff
@@ -110,10 +110,13 @@ def check_tcp_status(ip, port, timeout)
         data['status'] = 'unhealthy'
       end
       data
+<<<<<<< 822939a96818996084bfc7b1fd436f4c2573444b
```

Comment:
NB: this is just in the commit history, not in the complete branch diff, so it just looks weird, but it'll be fine in the squash commit.